### PR TITLE
tech: add secondary Card variant

### PR DESCRIPTION
**Pull request by Youakeem**
## Describe your changes
- Add the secondary card variant
- Move the Card aside from header level to card level

## Justify why they are needed
Align with Figma

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by Youakeem** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4709?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4710** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4710?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4709** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4709?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4708** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4708?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4707** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4707?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4706** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4706?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4705** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4705?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4704** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4704?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4703** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4703?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Youakeem and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by alebedev — APPROVED** [review]
Looks good

File: packages/ui/src/components/Card/Card.css.ts [hunk 1]
```diff
@@ -38,7 +50,9 @@ export const cardHeader = style([
 ])
 
 export const cardAside = style({
-  position: 'absolute',
-  top: 0,
-  insetInlineEnd: 0,
+  // @ts-expect-error: Hack to make sure aside content is always absolute. Should probably rething how we achieve this
+  // The main reason right now is giving tooltip trigger an explicit relative position, which is overriding this rule
+  position: 'absolute !important',
```
[the remark this answers is not in the document]
Comment (Youakeem) [reply packages/ui/src/components/Card/Card.css.ts:55]: I think so! Card.Aside takes a child element and applies this css class to it, if that element also have some `position` styles then it's a specificity game. In Vanilla-extract almost all selectors have the same specificity (just a class) so it's up to precedence which we can never guarantee.

Having import here overwrites whatever other position is being set in the child element.

**Comment by Youakeem** [comment]
### Merge activity

* **Sep 19, 3:16 AM EDT**: @Youakeem started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4709).

**Pull request merged**